playbooks/robusta_playbooks/pod_investigator_enricher.py

```diff
@@ -133,25 +134,29 @@ def report_pod_issue(
     # find pods with issues
     pods_with_issue = [pod for pod in pods if detect_pod_issue(pod) == issue]
     pod_names = [pod.metadata.name for pod in pods_with_issue]
+
+    if len(pods_with_issue) < 1:
+        logging.debug(f"`pods_with_issue` for found for issue: {issue}")
+        return
+
     expected_pods = get_expected_replicas(event)
-    message_string = f"{len(pod_names)}/{expected_pods} pod(s) are in {issue} state. "
+    message_string = f"{len(pod_names)}/{expected_pods} pod(s) are not ready"
     resource = event.get_resource()
     if resource.kind == "Job":
-        message_string = f"{len(pod_names)} pod(s) are in {issue} state. "
+        message_string = f"{len(pod_names)} pod(s) are not ready"
 
     # no need to report here if len(pods) != expected_pods since there are mismatch enrichers
 
-    blocks: List[BaseBlock] = [MarkdownBlock(message_string)]
     # get blocks from specific pod issue
-    additional_blocks = get_pod_issue_blocks(pods_with_issue[0])
+    pod_issues_enrichments = get_pod_issue_enrichments(pods_with_issue[0])
 
-    if additional_blocks:
-        blocks.append(MarkdownBlock(f"\n\n*{pod_names[0]}* was picked for investigation\n"))
-        blocks.extend(additional_blocks)
-        event.add_enrichment(blocks)
+    if pod_issues_enrichments:
+        issue_message, issues_enrichments = pod_issues_enrichments
+        blocks: List[BaseBlock] = [MarkdownBlock(f"{message_string}. {issue_message}")]
```

Comment:
This should be added to description, and not as Markdown

Reply:
done